lib/validate.js

```diff
@@ -197,7 +197,7 @@ async function subExists (name, { client, models, me }) {
     sub = await models.sub.findUnique({ where: { name } })
   }
 
-  return !!sub && sub.userId !== Number(me?.id)
+  return !!sub
```

Comment:
See #904. If we want to allow anyone to unarchive territories (including the previous owner), this needs to be removed.